src/index.html

```diff
@@ -29,6 +29,759 @@
   <script rel="preload" src="/assets/env.js" as="script"></script>
   <script>window.git_build = '{git-hash}';</script>
   <script>window.git_branch = '{git-branch}';</script>
+  <META HTTP-EQUIV='content-type' CONTENT='text/html; charset=UTF-8'>
+    <style>
+        #zohoSupportWebToCase textarea,
+        #zohoSupportWebToCase input[type='text'],
```

Comment:
Is so long  inline css needed here, can we take to separate css file and then include it?